test/integration/RecipientTest.py

```diff
@@ -28,78 +29,100 @@ def test_search_and_pagination(self):
         account_id1 = response.id
-
+        self.assertTrue(account_id1, "Account1 ID not found")
+        
         payload = {"type": "bank-transfer", "currency": "EUR",
-                   "iban": "FR14 2004 1010 0505 0001 3M02 606", "country": "FR"}
-        response = self.client.recipient_account.create(recipient_id, payload)
+                   "iban": "FR14 2004 1010 0505 0001 3M02 606", "country": "FR", "primary": False}
+        response = TestHelper.createRecipientAccount(recipient_id, payload)
         account_id2 = response.id
+        self.assertTrue(account_id2, "Account2 ID not found")
 
-        response = self.client.recipient_account.delete(
+        # Test - Delete Recipient Account
+        response = self.client.recipient_account.delete(recipient_id, account_id1)
+        self.assertTrue(response,"Recipient Not Deleted")
+        
+        # Test - Find Recipient Account
+        response = self.client.recipient_account.find(
             recipient_id, account_id2)
-        self.assertTrue(response, True)
+        self.assertEqual(response.id, account_id2, "Search result doesn't match with the Test Account created")
 
-        response = self.client.recipient_account.find(
-            recipient_id, account_id1)
-        self.assertTrue(response.id, account_id1)
+        # Test - Update Recipient Account
+        response = self.client.recipient_account.update(recipient_id, account_id2, {"primary": True})
+        self.assertTrue(response.primary, "Account country not updated")
+
+        # Cleanup
+        response = self.client.recipient.delete(recipient_id)
+        self.assertTrue(response)
 
     def test_routeMinimum(self):
-        uuidString = str(uuid.uuid4())
-        payload = {"type": "individual", "firstName": "Tom", "lastName": "Jones",
-                   "email": "test.create" + uuidString + "@example.com",
-                   "address": {
-                        "street1": '123 Wolfstrasse',
-                        "city": 'Berlin',
-                        "country": 'DE',
-                        "postalCode": '123123'
-                    },
-                   "account": {
-                        "type": "bank-transfer", "currency": "EUR",
-                        "iban": "DE89 3704 0044 0532 0130 00", "country": "DE"}
-                        }
-        response = self.client.recipient.create(payload)
+        # Setup - Create recipient
+        response = TestHelper.createRecipient()
         recipient_id = response.id
         
         self.assertTrue(response.routeType == "sepa")
         self.assertTrue(response.routeMinimum == "3")
 
+        # Cleanup
         response = self.client.recipient.delete(recipient_id)
         self.assertTrue(response)
 
+    def test_recipient_payments(self):
+        payments = self.client.recipient.get_all_payments("R-JSU6rUKR4xb4wSMAyG7L8U")
+        self.assertEqual(payments[len(payments)-1].records,2)
+
+        try:
```

Comment:
Why try catch is needed for the offline payment but not for get all payments?

Reply:
Fixed the assertion to remove the need for it